save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/PermissionController.kt

```diff
@@ -61,16 +61,24 @@ class PermissionController(
     )
     fun getRole(@PathVariable organizationName: String,
                 @PathVariable projectName: String,
-                @RequestParam userName: String,
+                @RequestParam(required = false) userName: String?,
                 authentication: Authentication,
-    ): Mono<Role> = permissionService.findUserAndProject(userName, organizationName, projectName)
+    ): Mono<Role?> = permissionService.findUserAndProject(
```

Comment:
I do not think that it is safe to give everyone permission to `getRole` of any user of any organization...

Reply:
We check here if current user has read permissions in this project. So I am not sure what you mean.

Reply:
getRole - is an endpoint in a controller. It is available from outside for everyone, isn’t it?

Reply:
It is avaliale from outside, but don't we check permissions here and return an empty Mono if permissions are not gained?

Reply:
@petertrr please check if this endpoint is safe enough